src/Device/AmazfitGTR.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Runalyze Device List.
+ *
+ * (c) RUNALYZE <mail@runalyze.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Runalyze\Devices\Device;
+
+class AmazfitGTR extends AbstractDevice
+{
+    use AmazfitDeviceTrait;
+
+    public function getEnum()
+    {
+        return DeviceProfile::AMAZFIT_GTR;
```

Comment:
``` 
Fatal error:  Class 'Runalyze\Devices\Device\AmazfitGtr' not found in /home/travis/build/Runalyze/devices/vendor/runalyze/common/src/Enum/AbstractEnumFactoryTrait.php on line 42
```

The factory expects for `AMAZFIT_GTR` the class name `AmazfitGtr`. For using `AmazfitGTR` the enum should be `AMAZFIT_G_T_R` (as it's done for e.g. `DeviceProfile::GARMIN_VIVO_ACTIVE_H_R`). There are some more enums with the same issue.